### ISSUE TITLE
Running `lxc start` on a container with an autostart delay pauses `lxc` until delay expires

### ISSUE THREAD

**Issue by fwaggle** [lxc/lxd#5129]
# Required information

 * Distribution: Ubuntu
 * Distribution version: 18.04

## Output of "lxc info"

```
config:
  core.https_address: '[::]:8443'
  core.trust_password: true
api_extensions:
- storage_zfs_remove_snapshots
- container_host_shutdown_timeout
- container_stop_priority
- container_syscall_filtering
- auth_pki
- container_last_used_at
- etag
- patch
- usb_devices
- https_allowed_credentials
- image_compression_algorithm
- directory_manipulation
- container_cpu_time
- storage_zfs_use_refquota
- storage_lvm_mount_options
- network
- profile_usedby
- container_push
- container_exec_recording
- certificate_update
- container_exec_signal_handling
- gpu_devices
- container_image_properties
- migration_progress
- id_map
- network_firewall_filtering
- network_routes
- storage
- file_delete
- file_append
- network_dhcp_expiry
- storage_lvm_vg_rename
- storage_lvm_thinpool_rename
- network_vlan
- image_create_aliases
- container_stateless_copy
- container_only_migration
- storage_zfs_clone_copy
- unix_device_rename
- storage_lvm_use_thinpool
- storage_rsync_bwlimit
- network_vxlan_interface
- storage_btrfs_mount_options
- entity_description
- image_force_refresh
- storage_lvm_lv_resizing
- id_map_base
- file_symlinks
- container_push_target
- network_vlan_physical
- storage_images_delete
- container_edit_metadata
- container_snapshot_stateful_migration
- storage_driver_ceph
- storage_ceph_user_name
- resource_limits
- storage_volatile_initial_source
- storage_ceph_force_osd_reuse
- storage_block_filesystem_btrfs
- resources
- kernel_limits
- storage_api_volume_rename
- macaroon_authentication
- network_sriov
- console
- restrict_devlxd
- migration_pre_copy
- infiniband
- maas_network
- devlxd_events
- proxy
- network_dhcp_gateway
- file_get_symlink
- network_leases
- unix_device_hotplug
- storage_api_local_volume_handling
- operation_description
- clustering
- event_lifecycle
- storage_api_remote_volume_handling
- nvidia_runtime
- container_mount_propagation
- container_backup
- devlxd_images
- container_local_cross_pool_handling
- proxy_unix
- proxy_udp
- clustering_join
- proxy_tcp_udp_multi_port_handling
- network_state
- proxy_unix_dac_properties
- container_protection_delete
- unix_priv_drop
- pprof_http
- proxy_haproxy_protocol
- network_hwaddr
- proxy_nat
- network_nat_order
- container_full
- candid_authentication
- backup_compression
- candid_config
api_status: stable
api_version: "1.0"
auth: trusted
public: false
auth_methods:
- tls
environment:
  addresses:
  - 10.0.4.1:8443
  - 10.254.0.1:8443
  architectures:
  - x86_64
  - i686
  certificate: |
    -----BEGIN CERTIFICATE-----
    MIIFOzCCAyOgAwIBAgIRALal1BBKwPZElwip36HZ3gYwDQYJKoZIhvcNAQELBQAw
    MjEcMBoGA1UEChMTbGludXhjb250YWluZXJzLm9yZzESMBAGA1UEAwwJcm9vdEBs
    YWIxMB4XDTE4MDkxNDExMjM1MVoXDTI4MDkxMTExMjM1MVowMjEcMBoGA1UEChMT
    bGludXhjb250YWluZXJzLm9yZzESMBAGA1UEAwwJcm9vdEBsYWIxMIICIjANBgkq
    hkiG9w0BAQEFAAOCAg8AMIICCgKCAgEA0qUhmntmDBFaLdi1+XpKHqXIInKqWU5K
    83LWatJEV8UuoUbH+rTuZ0+yaNBcp7yq5sGRJepIqdQ2+jhqoJTxMDJ2WYfaLl+Z
    y5u2cKMo1cwzz7yR2MfLq53PlYH47OqlOwPaiYgejKcaa+sBRLLciLuKO6zxUCXF
    gpGexrIOzcREQ8K2zejL/RopjmNDHJgzeJsuEhdxIJjU7/j7W7D4Zbze4oTzeTWe
    2GhbqLOnQEI1FPLcUWWQ4Sai/PReWx+9KKJ+PSlIswviEBDhbExOOSLXrIgnwdO/
    tAHsR3trY/fZoJjcZUMdQR4HfY2LE13krXgvc9jwZRDM9VtQ8yIX/DCdneolh+K7
    GYhD4lEgZnjLtDXTFMTNj4y+XsXvJTF/gdqvTTvukMnPbfBjnWb+ZBVD56h7jGZb
    FhHd1HhmAfM6Fra4Z62/LlZ5KruZNPLjh63iUnwQ0dA4Q/svEyHRMPZXinMexQ5d
    xkqgWelG70Iv+RKI22oHmqxWV13pcFs3tPO0/9qZQ4S/73Eb5sBP5UVx1b24H6MN
    U3ZFAnLG/y/KL7WW1X9b4i48wpuXY64QZitOf7aWYST+oU5OuOymeougH8A8jjeh
    a3D8OE4iRrRCtNgXyjqZ6M/ArqttOQQ85vytLe7bZgRrYRx6HkBeRiocT13x4H2j
    PLjEu87yvsECAwEAAaNMMEowDgYDVR0PAQH/BAQDAgWgMBMGA1UdJQQMMAoGCCsG
    AQUFBwMBMAwGA1UdEwEB/wQCMAAwFQYDVR0RBA4wDIIEbGFiMYcECgAEATANBgkq
    hkiG9w0BAQsFAAOCAgEAtX0IGGCFxCK6e+iPnAxuJfa4gqvjz1E1G/fXCBzonhYa
    3zmBYRSkem3F4N+3PKAG2u17JqxVOZdVu4EOy/TgNjuF+/IFTfbuQSLGmrlDsY2F
    oYcGimnDx5o90yU9/VxioQAqB7epZ3O+Y6cN+Dv/OfCHsNry1xleojT9ZWUazWS/
    yKYqvShGjAxGSlnPJ8m0JZ0dvzU2FHdXlFZeTLgsiTlNq6cRbZUD23ITf/L/6GYB
    4E99DGj0B2Y0PBfVKbPAc6gabPCwJwt7Ht9W3WyvLhkMbunC6KdvjxuAKp/92/JL
    FWBZp1YKec+eBpAI+TfAA/XsYZfXLqyvS8Ix9CmWF3FyUwIys+y9JfwiyonJ/CAu
    fR7grriF39LzL9yntrGTV4+jUBQfoMYQNZM3npSEaplnbQbnaaN1KDECCclNxI3b
    PP55zMLnaE2mI+N0lqQUef9BlPTWeF/jbYidqpzGwMzqUAqeV15NuR2iW5d9R6kb
    PkEWHH9J9InYtOS0/hpX6ZHYn1GvntKdTo4XnhTqCp3Uk04lelFv6gvvJIEQd874
    EJJx59GO9B0/s+nprElengN5HacaOqdQCkTzWkvRKszP+QYZK5ZvTUBD08P9yV0x
    h0RAlPM+XXsohtWxDlhs80+KnR4LIB2ncejUfeXFJyOJ2xkHWdzbzJe/HXZTnyM=
    -----END CERTIFICATE-----
  certificate_fingerprint: ae4c843c66c143e6b6ddb18b54c5877e096494d647ef26f7f7f7eba3319a2880
  driver: lxc
  driver_version: 3.0.2
  kernel: Linux
  kernel_architecture: x86_64
  kernel_version: 4.15.0-36-generic
  server: lxd
  server_pid: 11575
  server_version: "3.5"
  storage: zfs
  storage_version: 0.7.5-1ubuntu16.3
  server_clustered: false
  server_name: lab1
```

# Issue description

If a container with an autostart delay is waiting on autostart, it cannot be manually started. Instead, any "start" operation hangs for the autostart delay.

# Steps to reproduce

 1. Set a container to have an autostart delay, below I use 20 seconds.
 2. Stop the container manually, you can start it without issue immediately.
 3. Restart LXD daemon or machine so that all containers will be restarted according to their autostart settings.
 4. Attempt to start the container manually, the process will hang until the autostart delay is finished.

Expected behaviour: The timer is cancelled and the container is started immediately.

```
$ lxc config set test2 boot.autostart.delay 20
$ lxc stop test2
$ lxc list 
+-------+---------+---------------------+------+------------+-----------+
| NAME  |  STATE  |        IPV4         | IPV6 |    TYPE    | SNAPSHOTS |
+-------+---------+---------------------+------+------------+-----------+
| test1 | RUNNING | 10.254.1.187 (eth0) |      | PERSISTENT |           |
+-------+---------+---------------------+------+------------+-----------+
| test2 | STOPPED |                     |      | PERSISTENT |           |
+-------+---------+---------------------+------+------------+-----------+
$ time lxc start test2

real	0m0.556s
user	0m0.025s
sys	0m0.004s
$ lxc list 
+-------+---------+---------------------+------+------------+-----------+
| NAME  |  STATE  |        IPV4         | IPV6 |    TYPE    | SNAPSHOTS |
+-------+---------+---------------------+------+------------+-----------+
| test1 | RUNNING | 10.254.1.187 (eth0) |      | PERSISTENT |           |
+-------+---------+---------------------+------+------------+-----------+
| test2 | RUNNING | 10.254.1.153 (eth0) |      | PERSISTENT |           |
+-------+---------+---------------------+------+------------+-----------+
$ sudo snap stop lxd
Stopped.
$ lxc list 
+-------+---------+---------------------+------+------------+-----------+
| NAME  |  STATE  |        IPV4         | IPV6 |    TYPE    | SNAPSHOTS |
+-------+---------+---------------------+------+------------+-----------+
| test1 | RUNNING | 10.254.1.187 (eth0) |      | PERSISTENT |           |
+-------+---------+---------------------+------+------------+-----------+
| test2 | STOPPED |                     |      | PERSISTENT |           |
+-------+---------+---------------------+------+------------+-----------+
$ time lxc start test2
Error: The container is already running

real	0m11.855s
user	0m0.025s
sys	0m0.013s
```

The lxc start command was run a bit late, but it seems it will pause the entire length of the timer if you run it as lxd is starting.

# lxc --monitor output

```
metadata:
  context: {}
  level: dbug
  message: 'New event listener: 613bcb1a-d1b6-4cfe-b4c8-7050c195d285'
timestamp: "2018-10-10T04:07:39.609520545Z"
type: logging


metadata:
  context:
    alias: "18.04"
    fp: d72ae2e5073f20450c5260e6f227484c23452a46c6bb553ffe6be55e48602bb4
    protocol: simplestreams
    server: https://cloud-images.ubuntu.com/releases
  level: dbug
  message: Processing image
timestamp: "2018-10-10T04:07:39.614506492Z"
type: logging


metadata:
  context: {}
  level: dbug
  message: Connecting to a remote simplestreams server
timestamp: "2018-10-10T04:07:39.614525511Z"
type: logging


metadata:
  context: {}
  level: dbug
  message: Unmounted ZFS storage volume for container "test1" on storage pool "default"
timestamp: "2018-10-10T04:07:39.6465907Z"
type: logging


metadata:
  context: {}
  level: dbug
  message: Mounting ZFS storage volume for container "test1" on storage pool "default"
timestamp: "2018-10-10T04:07:39.651438849Z"
type: logging


metadata:
  context: {}
  level: dbug
  message: Mounted ZFS storage volume for container "test1" on storage pool "default"
timestamp: "2018-10-10T04:07:39.653780821Z"
type: logging


metadata:
  context:
    action: start
    created: 2018-10-04 21:56:57 +0000 UTC
    ephemeral: "false"
    name: test1
    stateful: "false"
    used: 2018-10-10 04:06:17.06591137 +0000 UTC
  level: info
  message: Starting container
timestamp: "2018-10-10T04:07:39.653830593Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:39.669555172Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/containers/1/onstart
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:39.673102513Z"
type: logging


metadata:
  context: {}
  level: dbug
  message: Mounting ZFS storage volume for container "test1" on storage pool "default"
timestamp: "2018-10-10T04:07:39.674797056Z"
type: logging


metadata:
  context: {}
  level: dbug
  message: Mounted ZFS storage volume for container "test1" on storage pool "default"
timestamp: "2018-10-10T04:07:39.675079936Z"
type: logging


metadata:
  context: {}
  level: dbug
  message: 'Scheduler: container test1 started: re-balancing'
timestamp: "2018-10-10T04:07:39.678973282Z"
type: logging


metadata:
  context: {}
  level: dbug
  message: 'Scheduler: network: vethIY3UTM has been added: updating network priorities'
timestamp: "2018-10-10T04:07:39.744890464Z"
type: logging


metadata:
  action: container-started
  source: /1.0/containers/test1
timestamp: "2018-10-10T04:07:39.744745738Z"
type: lifecycle


metadata:
  context:
    action: start
    created: 2018-10-04 21:56:57 +0000 UTC
    ephemeral: "false"
    name: test1
    stateful: "false"
    used: 2018-10-10 04:06:17.06591137 +0000 UTC
  level: info
  message: Started container
timestamp: "2018-10-10T04:07:39.744722349Z"
type: logging


metadata:
  context: {}
  level: dbug
  message: 'Scheduler: network: veth9KB58L has been added: updating network priorities'
timestamp: "2018-10-10T04:07:39.793832632Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:40.111148675Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:40.124668726Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:40.62584611Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:40.627661135Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:41.130336517Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:41.134486543Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:41.63804568Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:41.642556936Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:42.143465838Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:42.145255121Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:42.655482199Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:42.668078102Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:43.174674992Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:43.179968927Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:43.683328679Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:43.694215669Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:44.198751735Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:44.209743473Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:44.414432964Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0/containers?recursion=2
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:44.416488458Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:44.715326098Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:44.725641476Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:45.22878295Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:45.232258732Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:45.670116816Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0/containers/test2
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:45.673116694Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0/events
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:45.680689118Z"
type: logging


metadata:
  context: {}
  level: dbug
  message: 'New event listener: 3254c6d7-93d6-4109-96cf-90befbd6b49b'
timestamp: "2018-10-10T04:07:45.680976365Z"
type: logging


metadata:
  context:
    ip: '@'
    method: PUT
    url: /1.0/containers/test2/state
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:45.681542416Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:45.733160786Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:45.735930899Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:46.239043362Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:46.251986714Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:46.755071736Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:46.765827382Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:47.268297219Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:47.270373901Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:47.771385313Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:47.773191056Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:48.274780713Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:48.278138793Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:48.77920634Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:48.782285915Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:49.285365052Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:49.306085489Z"
type: logging


metadata:
  context:
    image: d72ae2e5073f20450c5260e6f227484c23452a46c6bb553ffe6be55e48602bb4
  level: dbug
  message: Image already exists in the db
timestamp: "2018-10-10T04:07:49.425109162Z"
type: logging


metadata:
  context: {}
  level: dbug
  message: Image already exists on storage pool "default"
timestamp: "2018-10-10T04:07:49.425309485Z"
type: logging


metadata:
  context:
    fp: d72ae2e5073f20450c5260e6f227484c23452a46c6bb553ffe6be55e48602bb4
  level: dbug
  message: Already up to date
timestamp: "2018-10-10T04:07:49.425321036Z"
type: logging


metadata:
  context: {}
  level: info
  message: Done updating images
timestamp: "2018-10-10T04:07:49.425360601Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:49.808911849Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:49.813230743Z"
type: logging


metadata:
  context: {}
  level: info
  message: Done updating instance types
timestamp: "2018-10-10T04:07:50.265773482Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:50.314292022Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:50.316232603Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:50.818031801Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:50.820739318Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:51.324287006Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:51.327529622Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:51.83057536Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:51.834998255Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:52.337131316Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:52.339353304Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:52.842094762Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:52.854250383Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:53.359272436Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:53.362218724Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:53.865018579Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:53.867526523Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:54.368733981Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:54.371686741Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:54.875702515Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:54.879349996Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:55.38099785Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:55.385127827Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:55.888731011Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:55.892686455Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:56.396387515Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:56.400175928Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:56.902692634Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:56.904633392Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:57.406291836Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:57.40882752Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:57.913289553Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:57.917964708Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:58.421523168Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:58.425026824Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:58.927240966Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:58.930098991Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:59.433550583Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:59.437254921Z"
type: logging


metadata:
  context:
    name: test2
  level: warn
  message: Unable to update backup.yaml at this time
timestamp: "2018-10-10T04:07:59.8053263Z"
type: logging


metadata:
  action: container-updated
  source: /1.0/containers/test2
timestamp: "2018-10-10T04:07:59.805358391Z"
type: lifecycle


metadata:
  context: {}
  level: dbug
  message: Mounting ZFS storage volume for container "test2" on storage pool "default"
timestamp: "2018-10-10T04:07:59.808266188Z"
type: logging


metadata:
  context: {}
  level: dbug
  message: Mounted ZFS storage volume for container "test2" on storage pool "default"
timestamp: "2018-10-10T04:07:59.810704528Z"
type: logging


metadata:
  context: {}
  level: dbug
  message: Unmounting ZFS storage volume for container "test2" on storage pool "default"
timestamp: "2018-10-10T04:07:59.812442667Z"
type: logging


metadata:
  context: {}
  level: dbug
  message: Unmounted ZFS storage volume for container "test2" on storage pool "default"
timestamp: "2018-10-10T04:07:59.832200985Z"
type: logging


metadata:
  context: {}
  level: dbug
  message: Mounting ZFS storage volume for container "test2" on storage pool "default"
timestamp: "2018-10-10T04:07:59.86894781Z"
type: logging


metadata:
  context: {}
  level: dbug
  message: Mounted ZFS storage volume for container "test2" on storage pool "default"
timestamp: "2018-10-10T04:07:59.871888201Z"
type: logging


metadata:
  context:
    action: start
    created: 2018-10-09 08:34:31 +0000 UTC
    ephemeral: "false"
    name: test2
    stateful: "false"
    used: 2018-10-10 04:06:37.902256086 +0000 UTC
  level: info
  message: Starting container
timestamp: "2018-10-10T04:07:59.871939902Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:59.888211365Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/containers/2/onstart
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:59.890594346Z"
type: logging


metadata:
  context: {}
  level: dbug
  message: Mounting ZFS storage volume for container "test2" on storage pool "default"
timestamp: "2018-10-10T04:07:59.901889792Z"
type: logging


metadata:
  context: {}
  level: dbug
  message: Mounted ZFS storage volume for container "test2" on storage pool "default"
timestamp: "2018-10-10T04:07:59.902179379Z"
type: logging


metadata:
  context: {}
  level: dbug
  message: 'Scheduler: container test2 started: re-balancing'
timestamp: "2018-10-10T04:07:59.905862916Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:59.944660487Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:07:59.967977714Z"
type: logging


metadata:
  action: container-started
  source: /1.0/containers/test2
timestamp: "2018-10-10T04:07:59.975608379Z"
type: lifecycle


metadata:
  context:
    action: start
    created: 2018-10-09 08:34:31 +0000 UTC
    ephemeral: "false"
    name: test2
    stateful: "false"
    used: 2018-10-10 04:06:37.902256086 +0000 UTC
  level: info
  message: Started container
timestamp: "2018-10-10T04:07:59.975586743Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:00.473624713Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:00.488188166Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:00.996749384Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:01.010312993Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:01.511685954Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:01.514581147Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:02.020504419Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:02.023053279Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:02.532935144Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:02.545516853Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:03.05214866Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:03.062683033Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:03.564541741Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:03.5670013Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:04.078234092Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:04.088995896Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:04.592223303Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:04.595956682Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:05.097447069Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:05.100345307Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:05.603947329Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:05.614746231Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:06.120159149Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:06.122496592Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:06.624076005Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:06.627001148Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:07.130310558Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:07.1346977Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:07.63929477Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:07.648965617Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:08.152223417Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:08.162936191Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:08.665851497Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:08.669528027Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:09.17315523Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:09.185785684Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:09.688634557Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:09.691580864Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:10.195090943Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:10.198875526Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:10.701804342Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:10.714469519Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:11.219760302Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:11.223877536Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:11.727742171Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:11.731755897Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:12.234701214Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:12.238744441Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:12.742019698Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:12.753206045Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:13.258689528Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:13.261991329Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:13.763713793Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:13.767658363Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:14.270647188Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:14.274801339Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:14.779471288Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:14.782894755Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:15.285672683Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:15.290505212Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:15.791843357Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:15.799194784Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:16.302499247Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:16.314087061Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:16.817631348Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:16.819836764Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:17.320907173Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:17.323867927Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:17.825182331Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:17.827051161Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:18.330607064Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:18.333858682Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:18.835303909Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:18.837172643Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:19.338730295Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:19.343387363Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:19.846491584Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:19.850664621Z"
type: logging


metadata:
  context: {}
  level: dbug
  message: 'New task operation: fe3e7b55-6bbc-4341-bc65-146344cc485c'
timestamp: "2018-10-10T04:08:20.011924301Z"
type: logging


metadata:
  class: task
  created_at: "2018-10-10T04:08:19.987434521Z"
  description: Starting container
  err: ""
  id: fe3e7b55-6bbc-4341-bc65-146344cc485c
  may_cancel: false
  metadata: null
  resources:
    containers:
    - /1.0/containers/test2
  status: Pending
  status_code: 105
  updated_at: "2018-10-10T04:08:19.987434521Z"
timestamp: "2018-10-10T04:08:20.01194864Z"
type: operation


metadata:
  context: {}
  level: dbug
  message: 'Started task operation: fe3e7b55-6bbc-4341-bc65-146344cc485c'
timestamp: "2018-10-10T04:08:20.012029033Z"
type: logging


metadata:
  class: task
  created_at: "2018-10-10T04:08:19.987434521Z"
  description: Starting container
  err: ""
  id: fe3e7b55-6bbc-4341-bc65-146344cc485c
  may_cancel: false
  metadata: null
  resources:
    containers:
    - /1.0/containers/test2
  status: Running
  status_code: 103
  updated_at: "2018-10-10T04:08:19.987434521Z"
timestamp: "2018-10-10T04:08:20.012038004Z"
type: operation


metadata:
  context: {}
  level: dbug
  message: 'Failure for task operation: fe3e7b55-6bbc-4341-bc65-146344cc485c: The
    container is already running'
timestamp: "2018-10-10T04:08:20.013575477Z"
type: logging


metadata:
  class: task
  created_at: "2018-10-10T04:08:19.987434521Z"
  description: Starting container
  err: The container is already running
  id: fe3e7b55-6bbc-4341-bc65-146344cc485c
  may_cancel: false
  metadata: null
  resources:
    containers:
    - /1.0/containers/test2
  status: Failure
  status_code: 400
  updated_at: "2018-10-10T04:08:19.987434521Z"
timestamp: "2018-10-10T04:08:20.013588003Z"
type: operation


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0/operations/fe3e7b55-6bbc-4341-bc65-146344cc485c
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:20.015088093Z"
type: logging


metadata:
  context: {}
  level: dbug
  message: 'Disconnected event listener: 3254c6d7-93d6-4109-96cf-90befbd6b49b'
timestamp: "2018-10-10T04:08:20.016261449Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /1.0
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:20.354754544Z"
type: logging


metadata:
  context:
    ip: '@'
    method: GET
    url: /internal/ready
  level: dbug
  message: handling
timestamp: "2018-10-10T04:08:20.358611595Z"
type: logging
```

**Comment by stgraber** [comment]
The logic in place is actually that all container state changes are held until the daemon is fully done processing all bootup actions, which includes starting boot time containers, updating images, expiring logs, ...

So it's not that your `lxc start` got to wait on that particular container, it got to wait for that entire part of the API to become available.
